oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/otlp/OpenTelemetryMetricRequestProcessor.java

```diff
@@ -334,4 +339,59 @@ private Stream<? extends Metric> adaptMetrics(
         }
         throw new UnsupportedOperationException("Unsupported type");
     }
+
+    public static String anyValueToString(AnyValue value) {
+        if (value.hasStringValue()) {
+            return value.getStringValue();
+        } else if (value.hasBoolValue()) {
+            return Boolean.toString(value.getBoolValue());
+        } else if (value.hasIntValue()) {
+            return Long.toString(value.getIntValue());
+        } else if (value.hasDoubleValue()) {
+            return Double.toString(value.getDoubleValue());
+        } else if (value.hasArrayValue()) {
+            return arrayValueToString(value.getArrayValue());
+        } else if (value.hasKvlistValue()) {
+            return keyValueListToString(value.getKvlistValue());
+        } else if (value.hasBytesValue()) {
+            return Base64.getEncoder().encodeToString(value.getBytesValue().toByteArray());
```

Comment:
It doesn't feel like a particularly appropriate way to handle this, but I'm not sure what to do about it.

Reply:
We only should take care of values meaningful.
Don't cover other cases if you don't know what to do with it.